File: javascript/p005.js
function gcd(a, b) {
	while (b != 0) {
		c = a % b;
		a = b;
		b = c;
	}
	return a;
}

function main() {
	let count = 1;
	for (let j = 1; j <= 20; j++) {
		count *= Math.floor(i / gcd(i, count));
	};
	return count;
};
console.log(main());

File: ruby/p005.rb
def gcd(a, b) 
	while b != 0 do
		c = a % b
		a = b
		b = c
	end
	return a
end

def main() 
	count = 1
	for i in 1..20 do
		count = count * (i / gcd(i, count).floor())
	end
	return count

end

puts main()